nequip/train/trainer.py

```diff
@@ -711,6 +714,7 @@ def init(self):
         self.init_objects()
 
         self._initialized = True
+        self.cumulative_wall = 0
```

Comment:
this doesn't get set again on restart?

Reply:
restart read it from the state_dict.

Reply:
but isn't `init()` called by `train()` after that?